ai/discord.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Discord bot"
+description: "Add a bot to your Discord server that answers questions based on your documentation."
+keywords: ["Discord", "bot", "Q&A"]
+---
+
+<Info>
+  Discord integrations are available on [Pro and Custom plans](https://mintlify.com/pricing?ref=discord) with access to the Mintlify assistant.
+</Info>
+
+Bring the knowledge in your documentation to your Discord server with the Mintlify assistant. The assistant powers a bot that answers user questions with information from your documentation. The bot searches your documentation to provide accurate, cited responses.
+
+The Discord bot responds to questions when a user mentions it with the `@` symbol.
+
+Each message sent by the Discord bot counts toward your plan's message allowance.
+
+## Add the Discord bot to your server
+
+<Note>
+  You must have Manage Server permissions in Discord to add the bot to your server.
```

Comment:
is this grammatically accurate? i was a little confused reading this off the bat.
```suggestion
  You must have "Manage Server" permissions in Discord to add the bot to your server.
```

Reply:
This is from the Discord UI. I think adding quotes helps make it more clear, but I'll take a look at ways to rewrite too.

Reply:
we could bold it too